src/benchmarks/micro/runtime/Math/Functions/Double/Acosh.cs

```diff
@@ -35,4 +35,4 @@ public static void AcoshTest()
             }
         }
     }
-}
```

Comment:
nit: please try to avoid changing end of the file (it adds a noise to the code review, but does not provide too much value)

Reply:
Sorry, will keep that it mind the next time.